File: packages/jsii-pacmak/test/expected.java/java/org/jsii/tests/calculator/JSObjectLiteralToNative.java
package org.jsii.tests.calculator;
@org.jsii.Jsii(module = org.jsii.tests.calculator.$Module.class, fqn = "jsii$jsii_calc$.JSObjectLiteralToNative")
public class JSObjectLiteralToNative extends org.jsii.JsiiObject {
    protected JSObjectLiteralToNative(final org.jsii.JsiiObject.InitializationMode mode) {
        super(mode);
    }
    public JSObjectLiteralToNative() {
        super(org.jsii.JsiiObject.InitializationMode.Jsii);
        org.jsii.JsiiEngine.getInstance().createNewObject(this);
    }
    public org.jsii.tests.calculator.JSObjectLiteralToNativeClass returnLiteral() {
        return this.jsiiCall("returnLiteral", org.jsii.tests.calculator.JSObjectLiteralToNativeClass.class);
    }
}

File: packages/jsii-dotnet-runtime/test/AWS.Jsii.Tests.Calculator/JSObjectLiteralToNative.cs
using AWS.Jsii.Runtime.Deputy;

namespace AWS.Jsii.Tests.Calculator
{
    [JsiiClass("jsii-calc", "jsii$jsii_calc$.JSObjectLiteralToNative", "[]")]
    public class JSObjectLiteralToNative : DeputyBase
    {
        public JSObjectLiteralToNative(): base(new DeputyProps(new object[]{}))
        {
        }

        protected JSObjectLiteralToNative(ByRefValue reference): base(reference)
        {
        }

        protected JSObjectLiteralToNative(DeputyProps props): base(props)
        {
        }

        [JsiiMethod("returnLiteral", "{\"fqn\":\"jsii$jsii_calc$.JSObjectLiteralToNativeClass\"}", "[]")]
        public virtual JSObjectLiteralToNativeClass ReturnLiteral()
        {
            return InvokeMethod<JSObjectLiteralToNativeClass>(new object[]{});
        }
    }
}